README.md

```diff
@@ -1,9 +1,20 @@
 # System-Design
 
-1. DNS:
-    * <a href="https://www.youtube.com/watch?v=mpQZVYPuDGU&ab_channel=PowerCertAnimatedVideos" target="_blank">What is DNS and how does it work?</a>
+System Design problems are very open ended in terms like you are given a problem and based on  the requirement you need to come up with the possible solution which can scale.
 
-2. CAP Theorem:
-    * [A plain english introduction to CAP Theorem](http://ksat.me/a-plain-english-introduction-to-cap-theorem)
-	
+This is generally asked from experienced candidates , usually 2-3 years experienced. 
+
+
+## Resources
+
+* System Design Primer Github :  <a href="https://github.com/donnemartin/system-design-primer"> System Design </a>
 
+
+* <a href="https://github.com/jeffrey-xiao/papers/blob/master/textbooks/designing-data-intensive-applications.pdf">Designing Data Intensive Applications</a> -- Holy Book for System Design 
```

Comment:
- keep links on the right, info on the left as per format. don't change on alternate points. or better, make the text a hyperlink like I used in the already added resources

- use a common delimiter => **:** between the above two rather than **"--"** to keep things looks consistent.

Reply:
@ayush571995